privacy_evaluator/datasets/dataset.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import torch
+from torch.utils.data import DataLoader
+from typing import Tuple
+
+
+class Dataset:
+    """Dataset base class."""
+
+    DATA_ROOT = "./data"
+
+    @classmethod
+    def numpy(cls, one_hot_encode: bool = False) -> Tuple[np.ndarray, ...]:
+        """Loads train and test dataset as a numpy arrays.
+
+        :param one_hot_encode: If data should be one-hot-encoded.
+        :return: Train and Test data and labels as numpy arrays.
+        """
+        raise NotImplementedError(
+            "Method 'numpy()' needs to be implemented in subclass."
+        )
+
+    @classmethod
+    def pytorch_loader(
+        cls,
+        train_batch_size: int = 128,
```

Comment:
would the batch size not be something that could be kept variable? Such that a user can specify a different batch size by calling the method?

Reply:
Yes, that is possible already possible. You can call for example `pytoch_loader(train_batch_size=32, test_batch_size=32)`. `128` is just a default value. Or didn't I understand the question correctly?

Reply:
Perfect, thank you! No, that was the question.